python/ql/src/Statements/UnreachableCode.ql

```diff
@@ -13,6 +13,18 @@
 
 import python
 
+private Name variableInvolvedInComparison (Compare c) {
+    result = c.getLeft().getASubExpression*() or
+    result = c.getAComparator().getASubExpression*()
+}
+
+private predicate allInvolvedVariablesAreInvolvedInPrevious(Compare c, Compare prevc) {
+    forall (Name n | 
+        n = variableInvolvedInComparison(c) | 
+        (exists (Name n2, Variable v | 
```

Comment:
The parentheses around the `exists` statement can be removed.

Reply:
Done